universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -107,6 +110,14 @@ public void putLong(Path path, String value) {
     }
   }
 
+  public void putList(Path path, ImmutableList<String> value) {
+    if (value.isEmpty()) {
+      putNull(path);
+    } else {
+      put(path, LIST_JOINER.join(value));
```

Comment:
Suggestion (blocking): validate that none of the values contain a backtick

Reply:
Added logic - I threw a `RuntimeException`, but is there a better option?

Reply:
I don't think we should throw runtime exceptions for things the admin _could_ do.